orderbook/src/api/get_fee_and_quote.rs

```diff
@@ -53,117 +57,25 @@ struct BuyQuery {
     buy_amount_after_fee: U256,
 }
 
+impl From<BuyQuery> for OrderQuoteRequest {
+    fn from(query: BuyQuery) -> Self {
+        let side = OrderQuoteSide::Buy {
+            buy_amount_after_fee: query.buy_amount_after_fee,
+        };
+        OrderQuoteRequest::new(query.sell_token, query.buy_token, side)
+    }
+}
+
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
-struct BuyResponse {
+pub struct BuyResponse {
```

Comment:
I think these structs (this one and `SellResponse`) aren't used at all anymore in favour of `FeeResponse`.